streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -86,7 +88,7 @@ public Collection<StandbyTask> getUpdatingStandbyTasks() {
         }
 
         public boolean onlyStandbyTasksLeft() {
-            return !updatingTasks.isEmpty() && updatingTasks.values().stream().noneMatch(Task::isActive);
+            return !updatingTasks.isEmpty() && updatingTasks.values().stream().allMatch(t -> !t.isActive());
```

Comment:
Seems it's changed back lol :P we can change this again if we want later.

Reply:
Oh no! That was not my intention! Let's change it back!